config/locales/en.yml

```diff
@@ -191,6 +191,8 @@ en:
         label: Recipient region
       sector:
         label: Focus area
```

Comment:
Now we have 2 types of "focus area", I wonder if we can give a better name to the old value more generally throughout this work to give the reader more clarity.

Reply:
Agree, one for content design I think?

Reply:
Yes, I did cover this with @deberny and we felt it best to use something and the circle back in content review?

Reply:
Happy with that 👍